spec/arrays.dd

```diff
@@ -456,18 +456,30 @@ double[][] matrix;
 
 $(SPEC_RUNNABLE_EXAMPLE_COMPILE
 ---------
-double[3][3] matrix;
+double[6][3] matrix;
 ---------
 )
 
-        $(P declares a rectangular matrix with 3 rows and 3 columns, all contiguously in memory. In
+        $(P declares a rectangular matrix with 3 rows and 6 columns, all contiguously in memory. In
         other languages, this would be called a multidimensional array and be declared as:
         )
 
 ---------
-double matrix[3,3];
+double matrix[3,6];
```

Comment:
Yes. I think it would be best to remove that whole sentence (from "In other languages..." upto and including this example). The "other language" is unspecified and the term "multidimensional array" can be used in D as well. Mixing another language with a different syntax into this adds more confusion than clarity. Besides, since C does not check bounds, `double matrix[6][3]` and `double matrix[3][6]` and even `double matrix[18]` could be used interchangeably, they all return a pointer to the same size memory.

The origin of this sentence is unclear. It was last touched 11 years ago by Andrei when it was moved from somewhere else, I can't see from where.

Is removal of this sentence appropriate in this PR, or should it be a different one?